R/RT_functions.R

```diff
@@ -37,17 +37,14 @@ create_ticket <- function(award, requestor) {
                                  subject = subject,
                                  rt_base = 'https://support.nceas.ucsb.edu/rt')
   
-  if (!grepl('created', httr::content(ticket))) {
-    out <- sprintf('I failed to create a ticket for award: %s, from requestor: %s', award, requestor)
-    slackr::slackr_bot(out)
-    return('rt_ticket_create_error')
+  #check to see if the object ticket is created successfully
+  if(!exists("ticket")) {
```

Comment:
Hi Jasmine, are you testing for the existence of the string "ticket" here? I'm not quite following that. Perhaps that should be unquoted to be the ticket variable?

Reply:
Thanks Chris! Will look at this as soon as I can.

Reply:
finally looked at it. Looks like it was a mistake on my part and I have fixed it in the updated commit. Thanks for catching that!

Reply:
gotcha! 👍